src/coreclr/tests/src/readytorun/crossboundarylayout/a/a.cs

```diff
@@ -30,6 +30,11 @@ public static void ReportTestFailure(string test, object o, ref int failCount)
             Console.WriteLine(test);
             s_testFailObj = o;
             failCount++;
+            while (true)
+            {
+                Thread.Sleep(1000);
+            s_testFailObj = o;
```

Comment:
_nit_: indent

Reply:
Ah, that was temporary logic in the test to make it easier to debug. I'll delete the new code.